### ISSUE TITLE
download_rr_archive: timeout in scrapping.py

### ISSUE THREAD

**Issue by pchalasani** [MatthieuBizien/roam-to-git#19]
I changed the number of iterations at line 103 to 500, but it still times out. Refreshing my db doesn't take that long, so maybe there's something else wrong. Or is it simply due to the long DB load-times these days in Roam? Is this supposed to work and I'm missing something?
Thanks for making this script available: it's awesome to see how puppeteer can be used via Python!

```
2020-05-10 11:30:43.538 | DEBUG    | roam_to_git.scrapping:download_rr_archive:54 - Creating browser
2020-05-10 11:30:44.320 | DEBUG    | roam_to_git.scrapping:_download_rr_archive:91 - Configure downloads to /var/folders/dn/6sb0pjt535qfrtnc1jyvk9lh0000gn/T/tmp11gi5ais
2020-05-10 11:30:44.354 | DEBUG    | roam_to_git.scrapping:signin:181 - Opening signin page
2020-05-10 11:30:49.194 | DEBUG    | roam_to_git.scrapping:signin:185 - Fill email 'pchalasani@xaipient.com'
2020-05-10 11:30:50.540 | DEBUG    | roam_to_git.scrapping:signin:190 - Fill password
2020-05-10 11:30:51.395 | DEBUG    | roam_to_git.scrapping:signin:195 - Click on sign-in
2020-05-10 11:30:53.648 | DEBUG    | roam_to_git.scrapping:go_to_database:205 - Load database from url 'https://roamresearch.com/#/app/xxxxxxx'
2020-05-10 11:30:53.704 | DEBUG    | roam_to_git.scrapping:_download_rr_archive:101 - Wait for interface to load
2020-05-10 11:48:01.981 | DEBUG    | roam_to_git.scrapping:download_rr_archive:75 - Closing browser json
2020-05-10 11:48:02.024 | DEBUG    | roam_to_git.scrapping:download_rr_archive:77 - Closed browser json
Traceback (most recent call last):
  File "./roamresearch.py", line 61, in <module>
    main()
  File "./roamresearch.py", line 49, in main
    asyncio.get_event_loop().run_until_complete(atask)
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/asyncio/base_events.py", line 584, in run_until_complete
    return future.result()
  File "/Users/pchalasani/Dropbox-personal/GitForks/Roam/roam-to-git/roam_to_git/scrapping.py", line 68, in download_rr_archive
    return await _download_rr_archive(document, output_type, output_directory, config)
  File "/Users/pchalasani/Dropbox-personal/GitForks/Roam/roam-to-git/roam_to_git/scrapping.py", line 119, in _download_rr_archive
    assert dot_button is not None, "All roads leads to Roam, but that one is too long. Try " \
AssertionError: All roads leads to Roam, but that one is too long. Try again when Roam servers are faster.
```

**Comment by MatthieuBizien** [comment]
Roam scrapping is hard to stabilize, and I don't know if this is on Roam side. Could you try with `--debug`?

**Comment by tfiers** [comment]
I had this same issue (also same as #22 and #27, probably): after a few days of working perfectly, all runs started failing, with `assert dot_button is not None, "All roads leads to Roam, but that one is too long. Try again when Roam servers are faster."`

I ran locally with `--debug` as per Matthieu's suggestion, and found out that the **problem was at login**: "The password is invalid or the user does not have a password.".

I did a password reset (actually entering the same password as I had earlier), and it started working again.

(I hypothesise this problem started when Roam added _Sign In with Google_, in the process erasing existing passwords).

Btw @MatthieuBizien, thanks again for this beautiful tool.
